### ISSUE TITLE
Server functionality for hiding api keys

### ISSUE THREAD

**Issue by SeanCassiere** [SeanCassiere/metro-quest#36]
Need server functionality to hide sensitive API keys.

Store the keys provide the keys on Netlify using serverless functions. [https://www.netlify.com/products/functions/](https://www.netlify.com/products/functions/)

**Comment by SeanCassiere** [comment]
@vinula98 @ArchanaVL99 @YomalNimeshka 

You'll need to `yarn install` on the dev branch once the PR for this is merged into the `dev` branch.

**Comment by SeanCassiere** [comment]
@vinula98 First you'll need to add the Stripe Key into a `.env` file. The template for this file is in `.env.example`.

Then, you'll be able to fetch the Stripe API key using the URL provided from `utils/environment`.
```javascript
const { getStripeKey } = getServerUrls()
```
